invenio_deposit/config.py

```diff
@@ -84,6 +84,24 @@
     ),
 )
 
+DEPOSIT_FORM_TEMPLATES = dict(
+    default='/static/node_modules/invenio-records-js/dist/'
```

Comment:
Shall we specify only the filename and use `'static'` endpoint to generate full URL?